# Request
uint8[] data
bool    publish
---
# Response
int32   bytesSent
